mirrord/layer/tests/common/mod.rs

```diff
@@ -1383,20 +1383,44 @@ pub fn get_env(
             MIRRORD_LAYER_INTPROXY_ADDR.to_string(),
             intproxy_addr.to_string(),
         ),
+        (
+            LayerConfig::RESOLVED_CONFIG_ENV.to_string(),
+            config.encode().unwrap(),
+        ),
+        #[cfg(target_os = "macos")]
         (
             "DYLD_INSERT_LIBRARIES".to_string(),
             dylib_path.to_str().unwrap().to_string(),
         ),
+        #[cfg(target_os = "linux")]
         (
             "LD_PRELOAD".to_string(),
             dylib_path.to_str().unwrap().to_string(),
         ),
+        #[cfg(all(target_os = "macos", target_arch = "aarch64"))]
         (
-            LayerConfig::RESOLVED_CONFIG_ENV.to_string(),
-            config.encode().unwrap(),
+            "MIRRORD_MACOS_ARM64_LIBRARY".to_string(),
+            arm64_dylib_path().to_str().unwrap().to_string(),
         ),
     ]
     .into_iter()
     .chain(extra_vars_owned)
     .collect()
 }
+
+#[cfg(all(target_os = "macos", target_arch = "aarch64"))]
```

Comment:
why not make it a `once` fixture like the existing `dylib_path`?
```suggestion
#[cfg(all(target_os = "macos", target_arch = "aarch64"))]
#[fixture]
#[once]
```

Reply:
I considered making it a fixture but didn't do so because it will require adding an new parameter for each tests (behind the same compilation condition here `macos` + `aarch64`). But if it's a cleaner option, I don't mind making it a fixture, since it is very similar to the `dylib_path` fixture.

Reply:
Ok, it's not that important. A `OnceLock` is fine, it also wouldn't be the end of the world if we didn't have a "once" mechanism, I mostly just wanted it to be consistent with the other dylib path we have.